cmd/cluster-controller/main.go

```diff
@@ -83,13 +84,14 @@ func main() {
 		fmt.Fprintln(os.Stderr, err)
 		os.Exit(1)
 	}
+	clusterConfig := kcpclienthelper.NewClusterConfig(config)
```

Comment:
Creating a separate `clusterConfig` here since we need the unmodified config for informers/listers. This will be removed after informers are scoped. https://github.com/kcp-dev/kcp/compare/main...varshaprasad96:kcp:wrap-apiresource-controller?expand=1#diff-03df9c971de272e090fac0bbcdb3f0bca9fbccc2f168aafff1e3cdf7e33eb915R100